Upstream: https://github.com/MariaDB/server/pull/44
Reason: Makes sure that tokuftdump is installed to the right path under cross

From fc4df415049271ef625783c31f2999e7d1bdc816 Mon Sep 17 00:00:00 2001
From: Niels Ole Salscheider <niels_ole@salscheider-online.de>
Date: Sun, 19 Apr 2015 12:40:00 +0200
Subject: [PATCH] tokuftdump: Install to ${INSTALL_BINDIR} instead of bin

---
 storage/tokudb/ft-index/tools/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/storage/tokudb/ft-index/tools/CMakeLists.txt b/storage/tokudb/ft-index/tools/CMakeLists.txt
index 4ed0cb4..71c44df 100644
--- a/storage/tokudb/ft-index/tools/CMakeLists.txt
+++ b/storage/tokudb/ft-index/tools/CMakeLists.txt
@@ -14,7 +14,7 @@ target_link_libraries(ftverify m)
 
 install(
   TARGETS tokuftdump
-  DESTINATION bin
+  DESTINATION ${INSTALL_BINDIR}
   COMPONENT Server
   )
 
-- 
2.3.3

